skrifa/src/color/mod.rs

```diff
@@ -1,6 +1,453 @@
-//! Color outlines.
+//! Drawing color glyphs.
+//!
+//! # Examples
+//! ## Retrieve the clip box of a COLRv1 glyph if it has one:
+//!
+//! ```
```

Comment:
Looks like the examples contain some errors. Usually cargo test catches these. I’m not sure why CI isn’t complaining here.

Reply:
Example compile checks should be fixed now and formatted a bit more cleanly.